helper/build_zips.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+PKG_NAME=mondoo
+if [[ ${VERSION} == "" ]]; then
+  echo "ERROR: Please define your version"
+  export VERSION=0.0.1
+fi
+
+echo "--------- Creating ZIP Package ${PKG_NAME}"
+
+# Create the base tgz:
+mkdir -p packages
+cd packages
+zip ${PKG_NAME}_${VERSION}_windows_amd64.zip ../mondoo.ps1 .
```

Comment:
Curious: what's the purpose of the trailing `.` at the end of this zip command? Running it locally didn't seem to have an effect.